cloud-nio/cloud-nio-spi/src/main/scala/cloud/nio/spi/UnixPath.scala

```diff
@@ -69,7 +69,7 @@ final private[spi] case class UnixPath(path: String) extends CharSequence {
 
   def isAbsolute: Boolean = UnixPath.isAbsolute(path)
 
-  def isEmpty: Boolean = path.isEmpty
+  override def isEmpty: Boolean = path.isEmpty
```

Comment:
Compiler was breaking without this, not related to PR

Reply:
Out of curiosity, what is your Java version?

I believe Cromwell is usually compiled with [Java 11](https://github.com/broadinstitute/cromwell/blob/a83ec6e76a4cedd0a8f29e29a41940bb6064140f/.github/set_up_cromwell_action/action.yml#L41-L46), and [`CharSequence::isEmpty`](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/lang/CharSequence.html#isEmpty()) is "Since: 15"

This override will need to be fixed, but maybe not right now?

Btw, I use [SDKMAN!](https://sdkman.io/usage#env) to keep all my Java SDKs under control. The JDK referenced by Cromwell's [config](https://github.com/broadinstitute/cromwell/blob/a83ec6e76a4cedd0a8f29e29a41940bb6064140f/.sdkmanrc#L3) isn't available for my M2 mac. I've just changed it on my machine to a newer 11 Temurin release.

Reply:
@harrismendell

> Have you had a moment to review the PR otherwise?

I've glanced at it, but I'm not an official committer at the moment. I've got multiple of own PRs [in the queue](https://github.com/broadinstitute/cromwell/pulls?q=is%3Aopen+is%3Apr+author%3Akshakir+-is%3Adraft+) to get reviewed 😅 

In the meantime I've just coalesced my commits into a fork and just compile and run that melange to keep moving.